### ISSUE TITLE
next steps

### ISSUE THREAD

**Issue by atherdon** [ChickenKyiv/loopback-tutor-intern-8#8]
We need to decide what we should do, while i cannot sit down with you and check your code 

options:
1) we can create a fast version of the [draft homepage](https://github.com/GroceriStar/frontend/issues/14). it'll be changed later, but in order to speed up development - this can be a good task
[not finished, but partly done code for homepage](https://github.com/GroceriStar/groceristar-static) from the previous intern

2) you can explore [this repository](https://github.com/GroceriStar/frontend/issues) yourself and make some minor changes. it's messy right now, but i'm working on make it better for you.

**Comment by chauhannishith** [comment]
Do you want me to start with this until I can find a solution to google login and re-verification(after changing email) part?

**Comment by chauhannishith** [comment]
The react is currently separate so there is no issue in that. There are only a few parts which do not work as planned, like google login(due to react and loopback being separate and also insufficient documentation or examples). I don't mind recreating this in jade if you want someone else to handle the react part at a later stage

**Comment by atherdon** [comment]
yes, loopback have outdated tutorials - agree with you. but it still rocks :) 
not a prob with google login for now. 
i think i need to take time and check it too. becuase i feel deep sorry about not replying to your messages fast. it's not cool, but hope for increase our collaboration soon. just working on artciles. maybe you saw them on medium.com...

i don't want that pages in jade, because this frontend approach(i mean with ejs/jade/pug) is really make code less readable. this is why i want to separate logic(api) with views(frontend).

i think our first step will be to move react app into separated repo and see how all that pages like login, forgot, email templates - can be included into groceristar project. or we can keep it here, as separated branch. not sure - tell me.

Btw, do you like fronted or backend more. i mean i have a lot of pending work related to backend - wortking with just loopback. but i want to update frontend at groceristar asap too - so tell me what do you want to do.

**Comment by chauhannishith** [comment]
I can work on both front and back together and create a basic frontend part so that you can try out things on the go and also it would be easier to design stuff faster.

A separate repository for react doesn't make much of a difference as you will constantly need it to test things out and it is better if all things are in one place rather than spread out over different repositories as if multiple people work on it then both will have to be fetched and pushed separately and people might make mistakes

**Comment by atherdon** [comment]
Agree/disagree :)
at first what i want is to make your react code is not related to current
project. so answer to a question - if we just grab react part and include
it to groceristar repo - will it work? maybe applying this changes will be
your next task?

need to go back to article - it will also give you more details about
future frontend work. will reply later


On 24-Apr-2018 at 01:53 PM, chauhannishith wrote:


I can work on both front and back together and create a basic frontend part
so that you can try out things on the go and also it would be easier to
design stuff faster.

A separate repository for react doesn't make much of a difference as you
will constantly need it to test things out and it is better if all things
are in one place rather than spread out over different repositories as if
multiple people work on it then both will have to be fetched and pushed
separately and people might make mistakes

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/ChickenKyiv/loopback-tutor-intern-8/issues/8#issuecomment-383938882>,
or mute the thread

**Comment by chauhannishith** [comment]
Yes, this same code will work with groceristar but will have to make some changes in the DB and links as in your project and also I am ready for anything you give me to do, front/back as it a bit rewarding to solve new challenges :)

**Comment by atherdon** [comment]
if you want to change a code and refresh your eyes, you can checkout a
main groceristar project and see what it needs to be done.
btw, read this:
https://medium.com/groceristar/groceristar-roadmap-intro-part-1-b2cabc59f789


On 24-Apr-2018 at 02:02 PM, chauhannishith wrote:


Yes, this same code will work with groceristar but will have to make some
changes in the DB and links as in your project and also I am ready for
anything you give me to do, front/back as it a bit rewarding to solve new
challenges :)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/ChickenKyiv/loopback-tutor-intern-8/issues/8#issuecomment-383942766>,
or mute the thread

**Comment by chauhannishith** [comment]
Is it ok if I use files of this page temporarily?[https://github.com/GroceriStar/groceristar-react-homepage-2](url)

**Comment by atherdon** [comment]
@chauhannishith yes, but this is not the latest version. i'm still prefer to separate things, because it's easy to manage for me. and jump between projects. but it's cool to see that you move it here

**Comment by atherdon** [comment]
latest version - i'm working on it, if you wish - i can assign few small tasks to you.
https://github.com/atherdon/bebe-grocery repo
netlify link: https://goofy-keller-15b71e.netlify.com/

**Comment by chauhannishith** [comment]
yes I can do some extra things on the side along with what I am doing now

**Comment by chauhannishith** [comment]
I contacted the forestAdmin support but haven't got any response yet so I can do some small tasks until either the forestadmin support responds or I can find some way to update the stripe transactions in our database

**Comment by atherdon** [comment]
i think maybe they( forest admin ) will roll over a big updates soon. because as i see - after getting cash they have a lot of activity. so maybe soon it'll work.
OR! maybe we need to create a clean expressjs server, that will connect to our mlab instance and also to forest admin. maybe this is just an Loopback issue...
